.github/workflows/remove-labels.yml

```diff
@@ -0,0 +1,34 @@
+name: Remove Labels
+
+on:
+  pull_request_branch:
+    types: [labeled]
+
+jobs:
+  label_cleanup:
+    runs-on: ubuntu-latest
+
+    if: |
+      (github.event_name == 'pull_request' && contains(github.event.pull_request.labels.*.name, 'ci:run')) ||
+      (github.event_name == 'pull_request' && contains(github.event.pull_request.labels.*.name, 'ci:test'))
```

Comment:
remove `gihub.event_name == pull_request` since we only trigger this workflow on a pull_request?